sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -111,7 +111,7 @@ export interface CertificateIssuer extends IssuerProperties {
 }
 
 // @public
-export type CertificateKeyCurveName = "P-256" | "P-384" | "P-521" | "P-256K";
+export type CertificateKeyCurveName = KnownJsonWebKeyCurveName | string;
```

Comment:
In https://github.com/Azure/azure-sdk-for-js/pull/12821#discussion_r542616576, we use just string whereas here we have a union of the known enum and string.

Which way do we want to go?

cc @joheredi

Reply:
This is a manual mistake. Fixed it in the next commit.